src/Integration.UnitTests/Helpers/ToolWindowServiceTests.cs

```diff
@@ -143,7 +143,7 @@ public void GetToolWindow_ReturnsCorrectType()
             SetupFindToolWindow(serviceProviderMock, uiShellMock, VSConstants.S_OK, new Guid(MyDummyToolWindow.GuidAsString), frameMock.Object);
 
             var testSubject = new ToolWindowService(serviceProviderMock.Object);
-            var result = testSubject.GetToolWindow<MyDummyToolWindow>();
+            var result = testSubject.GetToolWindow<MyDummyToolWindow, MyDummyToolWindow>();
```

Comment:
It would be better to introduce a new `IDummyToolWindow` interface and use that in the test i.e. make the test more accurately reflect how the product code is going to be used.